cli/src/command/generate_config.rs

```diff
@@ -179,10 +157,8 @@ impl ExonumCommand for GenerateConfig {
         let private_config = NodePrivateConfig {
             listen_address,
             external_address: self.peer_address.to_string(),
-            consensus_public_key,
-            consensus_secret_key: CONSENSUS_SECRET_KEY_NAME.into(),
-            service_public_key,
-            service_secret_key: SERVICE_SECRET_KEY_NAME.into(),
+            master_key_path: master_key_file_name.into(),
```

Comment:
Bad naming, it's actually file name, not path. I will rename it.

Reply:
Does it make sense to store name (which is always the same) in the config files? I suggest you to store full path to the file.

Reply:
You are right, this is certainly path, but in case of `generate-config` command the path is relative, so we use `MASTER_KEY_FILE_NAME`. I think it's covers most of the cases, but you can later specify absolute path to master key manually if you need it.

Reply:
Sorry, I just don't see any use in such field if it's always equal to `MASTER_KEY_FILE_NAME`. I see two options here:
1. Remove the field completely, the master key will always be in `MASTER_KEY_FILE_NAME`, we clearly state this in documentation.
2. Store actual file location (preferably absolute path, but can be relative if the former one is not possible), possibly allow user to change it (with an optional configuration parameter), and return the path to the file in `StandardResult` so the user can expand the command later without any problems.

Right now the code shows a bit weird behaviour between these two options: we have a field, but it always contain constant value.

Reply:
I choose second variant.